packages/codegen-ui-react/lib/forms/form-renderer-helper.ts

```diff
@@ -242,54 +242,136 @@ export const addFormAttributes = (component: StudioComponent | StudioComponentCh
         factory.createJsxExpression(undefined, resetValuesName),
       ),
     );
+    if (formMetadata.formActionType === 'update') {
+      attributes.push(
+        factory.createJsxAttribute(
+          factory.createIdentifier('isDisabled'),
+          factory.createJsxExpression(
+            undefined,
+            factory.createPrefixUnaryExpression(
+              SyntaxKind.ExclamationToken,
+              factory.createParenthesizedExpression(
+                factory.createBinaryExpression(
+                  factory.createIdentifier('id'),
```

Comment:
Can you help me understand where `id` and `myData` get defined?

Reply:
Ahhh nice catch! They get both defined at the top of the generated forms. I shouldn't be hardcoding the variable name as it uses the data model's name. Fixing in next rev